tunnel.go

```diff
@@ -4,12 +4,14 @@ import (
 	"context"
 	"net"
 	"net/http"
+	"time"
 
 	tunnel_client "github.com/ngrok/libngrok-go/internal/tunnel/client"
 )
 
 type Tunnel interface {
 	CloseWithContext(context.Context) error
+	Close() error
```

Comment:
This pair of functions could use a doc-comment to explain when you might want one or the other.